README.md

```diff
@@ -83,3 +83,47 @@ print(chat_history.messages)
 ### Vectorstore
```

Comment:
A couple of comments for the future:
* Make the VectorStore the first integration in the README
* Remove the outdated change log